Domain/ProductManagement/Product.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LenaInventoryManagementSystem.Domain.ProductManagement
+{
+    public class Product
+    {
+        private string _productName;
+        private double _price;
+        private int _quantity;
+        private int _maxItemsInStock;
+
+        public Product()
+        {
+            _maxItemsInStock = 150;
```

Comment:
`150` is a magic number. Usually, hard coded strings and numbers saved in constants so we better describe what those constants mean. Lets move this number to a constant please.